homeassistant/components/light/mqtt/__init__.py

```diff
@@ -0,0 +1,62 @@
+"""
```

Comment:
I mean I guess the loader accepts it, but should be allow having modules (directories) for platforms? CC @balloob

Reply:
This was suggested by @balloob in #18180:
> To keep code manageable we could move the files so we have light/mqtt/__init__.py and light/mqtt/json.py

Reply:
Ok then, that's good 👍